src/main/fc/init.c

```diff
@@ -365,7 +364,7 @@ void init(void)
 
     pgResetAll();
 
-#if defined(STM32H7) && defined(USE_SDCARD_SDIO) // H7 only for now, likely should be applied to F4/F7 too
+#if defined(USE_SDCARD_SDIO)
     sdioPinConfigure();
     SDIO_GPIO_Init();
```

Comment:
Turns out these conditionals were done in the wrong way - as it is, on F4 and F7 there is only one set of pins available for SDIO, so there is no need to configure pins, and no need for `sdioPinConfigure()`. :disappointed: 
This will need some extensive rework to make it work the right way. Just put the conditional on H7 back in, and we can fix this in another pull request. Sorry for that.

Reply:
No worries. Pushed again.